examples/bin/crypt_worker

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
+require_relative '../init'
+require_relative '../lib/cryptconverter'
+
+require 'temporal/worker'
+
+Dir[File.expand_path('../workflows/*.rb', __dir__)].each { |f| require f }
+Dir[File.expand_path('../activities/*.rb', __dir__)].each { |f| require f }
+Dir[File.expand_path('../middleware/*.rb', __dir__)].each { |f| require f }
+
+Temporal.configure do |config|
+  config.task_queue = 'crypt'
+  config.converter = Temporal::Client::Converter::Crypt.new(
```

Comment:
This isn't a change, the crypt converter in both java and go wrap the default converters. This isn't something where I can just add a payload converter to the list of the composite converter because here we need to nest calls. The crypt converter runs the default converter and then wraps those payloads when encrypting, and vice versa when decrypting.

Reply:
Go version: https://github.com/temporalio/samples-go/blob/master/cryptconverter/worker/main.go#L19